scripts/orchestrator.js

```diff
@@ -32,6 +32,7 @@ const orchestrator = async (tabUrl, websiteUrl, userId) => {
 		// TODO: create a UI element on the screen with CTA to login to Vibinex
 	}
 	if (urlObj[2] == 'github.com') {
+		addSignedOutIndicator(websiteUrl, 'github')
```

Comment:
From a design perspective, do we want an indicator on the top like this, or should we reuse the same element we use for other indicators on the bottom left? 

I think all indicators should be in one place for a consistent experience. We can debate whether it should all be in the top-centre or bottom-left.

Reply:
@tapishr Let me know what you think.

Reply:
How's this?

<img width="1176" alt="Screenshot 2023-11-21 at 12 37 12 PM" src="https://github.com/Alokit-Innovations/chrome-extension/assets/55079486/0dd2b974-c587-4cf2-80f2-f91e69125905">

Reply:
Done in [5881e60](https://github.com/Alokit-Innovations/chrome-extension/pull/86/commits/5881e608c255a258561c6640e6c0ea660cb6803e)

Reply:
It would be better if we could make it more prominent. Something like this:

Reply:
Done in [9548c2f](https://github.com/Alokit-Innovations/chrome-extension/pull/86/commits/9548c2fba4114c795f57c1af73396c549b89eac4)

The new indicator, it'll also open the log in screen in new tab as mentioned in epic

<img width="360" alt="Screenshot 2023-11-21 at 10 27 59 PM" src="https://github.com/Alokit-Innovations/chrome-extension/assets/55079486/d2d03aa4-34fe-41a7-8c4d-2f694164a9fa">